src/pages/YearPickerPage.js

```diff
@@ -70,9 +70,10 @@ class YearPickerPage extends React.Component {
      * @param {String} text
      */
     filterYearList(text) {
+        const searchText = text.replace(/[\D]/g, '');
```

Comment:
```suggestion
        const searchText = text.replace(/\D/g, '');
```

Just a minor suggestion